### DISCUSSION TITLE
Unable to compile when importing, otherwise it works fine.

### DISCUSSION THREAD

**Discussion by hadiimran00** [crytic/slither#1717] [Q&A]
![image](https://user-images.githubusercontent.com/111147135/223416430-6366e662-beff-41ca-8d37-44cef76e56ab.png)
Program does not run/compile when I import otherwise it works fine in VS code. HELP Please, a amateur here.

**Comment by TacitvsXI** [comment]
Hey, if you're unable to compile, it could be because the '@openzeppelin/contracts' library is not installed. Try running 'npm install @openzeppelin/contracts' or 'yarn add @openzeppelin/contracts'. It should work after installing the packages. Hope it will help:)

**Reply by hadiimran00** [reply]
> Hey, if you're unable to compile, it could be because the '@openzeppelin/contracts' library is not installed. Try running 'npm install @openzeppelin/contracts' or 'yarn add @openzeppelin/contracts'. It should work after installing the packages. Hope it will help:)

done already, doesn't work
